app/javascript/controllers/index.js

```diff
@@ -21,3 +21,6 @@ application.register("video-player", VideoPlayerController)
 
 import WordCloudController from "./word_cloud_controller"
 application.register("word-cloud", WordCloudController)
+
+import ReplaceController from "./replace_controller"
+application.register("replace", ReplaceController)
```

Comment:
This naming pattern is okay but it is open to change. If changed here, the view with this will also need to be changed. 

Advice: Typically Stimulus controllers represent abstract concepts or actions; thus `script_controller.js` is atypical.